src/components/SignIn/__snapshots__/index.test.js.snap

```diff
@@ -92,7 +92,7 @@ exports[`renders correctly 1`] = `
           >
             <input
               aria-label="password"
-              className=""
+              className={null}
```

Comment:
not updated

Reply:
I will make it in a separate PR, is that okay? this PR will be huge then :/

Reply:
make the function return "" as you said. done!